public/server/db.json

```
@@ -109,6 +109,57 @@
     },
     "ootd":{
         "definition": "Outfit Of The Day"
-    }
-    
+    },
+    "cmiiw":{
+        "definition": "Correct me if i'm wrong"
+    },
+    "b2w":{
+        "definition": "Back to work"
+    },
+    "afk":{
+        "definition": "Away from Keyboard"
+    },
+    "cydm":{
+        "definition": "Check your Direct Message"
+    },
+    "ofc":{
+        "definition": "Of course"
+    },
+    "ama":{
+        "definition": "Ask Me Anything"
+    },
+    "awol":{
+        "definition": "Absence without Leave",
```

I think the main definition is 'Absent Without Official leave'?
```suggestion
        "definition": "Absent Without Official Leave",
```